edi_sale_utm_oca/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "EDI Sale UTM",
+    "summary": """
+    Automatically set edi source on the sale order when it is created.
+    """,
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/edi-framework",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "maintainers": ["simahawk"],
+    "depends": ["sale_management", "edi_utm_oca"],
```

Comment:
```suggestion
    "depends": ["edi_sale_oca", "edi_utm_oca"],
```
We need the `origin_exchange_record_id` on Sale Order